pallets/ajuna-affiliates/src/lib.rs

```diff
@@ -18,7 +18,7 @@
 
 pub use pallet::*;
 
-#[cfg(any(test, feature = "runtime-benchmarks"))]
+#[cfg(any(test))]
```

Comment:
It is best to not enable this with the runtime-features flag to ensure that we don't use stuff in the benchmarks we are not supposed to.